src/server.js

```diff
@@ -14,3 +16,17 @@ export default async (state: { page: string }): string => {
 
     return doc({ html, state, css, cssIDs });
 };
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
can just not have the `next` arg?

Reply:
Yes `next` can be removed as an argument